pkg/controller/extensions/request/extensionrequest.go

```diff
@@ -64,6 +65,9 @@ var (
 	resultRequeue    = reconcile.Result{Requeue: true}
 	requeueOnSuccess = reconcile.Result{RequeueAfter: requeueAfterOnSuccess}
 	jobBackoff       = int32(0)
+
+	// PodImageNameEnvVar is the env variable for setting the image name used for the unpack/install process when debugging the main process
+	PodImageNameEnvVar = "UNPACK_IMAGE"
```

Comment:
i don't think this env var is used outside of this package, so this variable shouldn't be exported (start its name with a lowercase).

also, to be a bit cleaner, the name of the env var itself could probably be more describing the information it's collecting and less describing how it will be used.  What we are collecting here is the image of the running code, so perhaps calling it `EXTENSION_MANAGER_IMAGE` would be more expressive.  that opinion is a bit subjective i realize, but it does feel more clean to me.